makefile.shade

```diff
@@ -360,7 +360,7 @@ var buildTarget = "compile"
 
                 if (IsMono)
                 {
-                    Exec("build.sh", buildTarget, repo);
+                    Exec(Path.Combine(repo, "build.sh"), buildTarget, repo);
```

Comment:
I'm not sure that's a good idea - I'd rather not touch it since it behaves as expected there.